getting-started/pom.xml

```diff
@@ -46,6 +46,22 @@
             <artifactId>rest-assured</artifactId>
             <scope>test</scope>
         </dependency>
+
+<!--        <dependency>-->
+<!--            <groupId>io.quarkus</groupId>-->
+<!--            <artifactId>quarkus-minikube</artifactId>-->
+<!--        </dependency>-->
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-container-image-jib</artifactId>
+        </dependency>
+
+<!--        <dependency>-->
+<!--            <groupId>io.quarkus</groupId>-->
+<!--            <artifactId>quarkus-minikube-deployment</artifactId>-->
+<!--            <scope>provided</scope>-->
+<!--            <version>${quarkus-plugin.version}</version>-->
+<!--        </dependency>-->
```

Comment:
This doesn't look like something you want to merge :).

Reply:
Absolutely not :)